common/src/main/scala/explore/model/GraphQLClients.scala

```diff
@@ -9,31 +9,39 @@ import cats.syntax.all.*
 import clue.*
 import io.circe.Json
 import lucuma.schemas.*
+import org.http4s.Uri.Authority
+import org.http4s.Uri.Scheme
 import org.http4s.*
+import org.http4s.syntax.all.*
 import org.typelevel.log4cats.Logger
 import queries.schemas.*
 
 case class GraphQLClients[F[_]: Async: Parallel] protected (
   odb:           WebSocketClient[F, ObservationDB],
   preferencesDB: WebSocketClient[F, UserPreferencesDB],
-  itc:           TransactionalClient[F, ITC]
+  itc:           TransactionalClient[F, ITC],
+  sso:           WebSocketClient[F, SSO]
```

Comment:
Is it worth it to keep a web socket connection open to SSO in every instance of explore? Maybe a `TransactionalClient` is enough?

(Or are you using it temporarily while we cannot refresh the token in a TransactionalClient?)

Reply:
Yes I want to use `TransactionalClient` but we need to be able to set the auth token first